Documentation/operations/upgrade.rst

```diff
@@ -162,7 +169,7 @@ command line or by committing the values to a YAML file:
     .. parsed-literal::
 
       helm template |CHART_RELEASE| \\
-        --set keepDeprecatedProbes=true \\
+        --set upgradeCompatibility=1.X \\
```

Comment:
Will Helm properly fail if the user fails to replace this value?

Reply:
Yup:

```
$ helm template ./install/kubernetes/cilium --set upgradeCompatibility=1.X
Error: template: cilium/templates/cilium-configmap.yaml:18:7: executing "cilium/templates/cilium-configmap.yaml" at <semverCompare ">=1.8" (default "1.8" .Values.upgradeCompatibility)>: error calling semverCompare: Invalid Semantic Version

Use --debug flag to render out invalid YAML
```